src/main/java/net/atos/client/zgw/brc/exception/BrcRuntimeException.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.client.zgw.brc.exception;
+
+import net.atos.client.zgw.shared.exception.ZgwRuntimeException;
+
+public class BrcRuntimeException extends ZgwRuntimeException {
```

Comment:
Isn't this just RuntimeException or we want to have a hierarchy?

Reply:
yes,  indeed. for easier handling later on.